security/yubikey-configuration.md

```diff
@@ -136,11 +136,11 @@ This subkey will be used to pull private git repos via SSH and may be used to au
 This is interface has a unique design where you need to toggle things on and off to get the desired result.
 The default state shows **Sign Encrypt** active.
 
-![Sign Encrypt Default State](https://github.com/trussworks/Engineering-Playbook/blob/yubikey-guide/security/images/Yubikey-Sign-Encrypt.png "default state")
+![Sign Encrypt Default State](https://github.com/trussworks/Engineering-Playbook/tree/master/security/images/Yubikey-Sign-Encrypt.png "default state")
```

Comment:
Relative paths should work I _think_ ...

Reply:
yeah, I think ryan might have the right idea here...